client/admin/store/index.js

```diff
@@ -13,5 +16,6 @@ export default new Vuex.Store({
     contentRepo,
     programs
   },
-  plugins
+  plugins: [authPlugin(MUTATION_TYPE)],
```

Comment:
Consider tweaking plugin so it can receive any mutation / storage key / store key combination (and persist value).